include/internal/libfreenect2/depth_packet_processor.h

```diff
@@ -175,6 +175,25 @@ class OpenCLDepthPacketProcessor : public DepthPacketProcessor
 private:
   OpenCLDepthPacketProcessorImpl *impl_;
 };
+
 #endif // LIBFREENECT2_WITH_OPENCL_SUPPORT
+
+class DumpDepthPacketProcessor : public DepthPacketProcessor
```

Comment:
We'd like to do offline processing of the depth files in order to facilitate faster/easier algorithm refining and testing.

When you mention the tables, can I pull them once and keep them around or are they re-calculated whenever the kinect restarts?

And yes, I'm seeing ~3Mb / frame, so yes, I think your size estimate is correct.  However I'm not offput by those sizes, and I can successfully capture to disk at those rates w/o frame dropping.